pygitguardian/client.py

```diff
@@ -327,7 +327,7 @@ def multi_content_scan(
         """
         if len(documents) > MULTI_DOCUMENT_LIMIT:
             raise ValueError(
-                "too many documents submitted for scan (max={0})".format(
+                "too many documents submitted for scan (max={})".format(
```

Comment:
I am surprised that we don't use f-strings in all these situations ? Is there a reason for this ?

Reply:
Good question. I don't know why pyupgrade did not change this one. Just did it.